code/game/objects/items/stacks/sheets/leather.dm

```diff
@@ -83,6 +83,42 @@
 	icon_state = "weed_extract"
 	origin_tech = ""
 
+/obj/item/stack/sheet/animalhide/deer
+	name = "deer hide"
+	desc = "The skin of a dead deer."
+	singular_name = "deer hide piece"
+	icon_state = "sheet_deer"
+	origin_tech = ""
+
+/obj/item/deer_head
+	name = "deer head"
+	desc = "Not something you want to find in your bed in the morning."
+	icon = 'icons/obj/butchering_products.dmi'
+	icon_state = "deer_head"
+	origin_tech = ""
+
+/obj/item/deer_head/attackby(obj/item/W, mob/user)
+	.=..()
+
+	if(W.is_sharp() >= 1.2) //Actually sharp things are this sharp, yes
```

Comment:
I was meaning to make it into a define, but fell asleep and enjoyed christmas with my family